intg-appletv/tv.py

```diff
@@ -752,3 +825,25 @@ async def launch_app(self, app_name: str) -> ucapi.StatusCodes:
     async def app_switcher(self) -> ucapi.StatusCodes:
         """Press the TV/Control Center button two times to open the App Switcher."""
         await self._atv.remote_control.home(InputAction.DoubleTap)
+
+    @async_handle_atvlib_errors
+    async def set_output_device(self, device_name: str) -> ucapi.StatusCodes:
+        """Set output device selection."""
+        if device_name is None:
+            return ucapi.StatusCodes.BAD_REQUEST
+        device_entry = self._output_devices.get(device_name, [])
+        if device_entry is None:
+            _LOG.warning("Output device not found in the list %s (list : %s)", device_name, self.output_devices)
+            return ucapi.StatusCodes.BAD_REQUEST
+        output_devices = self._atv.audio.output_devices
+        if len(device_entry) == 0 and len(output_devices) == 0:
+            return ucapi.StatusCodes.OK
+        device_ids = []
+        for device in output_devices:
+            device_ids.append(device.identifier)
+        _LOG.debug("Removing output devices %s", device_ids)
+        await self._atv.audio.remove_output_devices(*device_ids)
+        if len(device_entry) == 0:
+            return ucapi.StatusCodes.OK
+        _LOG.debug("Setting output devices %s", device_entry)
+        await self._atv.audio.set_output_devices(*device_entry)
```

Comment:
Adding an additional airplay speaker (the first speaker to be added) causes audio interrupt on my side too in all cases (even when I do this from my iphone, so this is not related to the way we use the library)

In the future I hope that we will be able to have a dropdown multiselect in the UI :-) so that we won't have to make the diffs anymore